### DISCUSSION TITLE
Avalonia UI Neumorphism theme

### DISCUSSION THREAD

**Discussion by flarive** [AvaloniaCommunity/awesome-avalonia#61] [Show and tell]
Hi,

I'm working on a Avalonia Neumorphism theme implementation.
Could you add it to the list of contributions please ?

Neumorphism Avalonia
https://github.com/flarive/Neumorphism.Avalonia
Easy to use and customizable Neumorphism Design implementation for Avalonia UI framework.

https://github.com/flarive/Neumorphism.Avalonia/blob/main/Avalonia.Neumorphism.Demo.gif

Thanks a lot :)
Eviral

**Comment by OmidID** [comment]
Hello. Looks cool. Please create a PR. THANKS

**Comment by wosiyourfather** [comment]
0.0
